Packs/Bitbucket/Integrations/Bitbucket/Bitbucket.py

```diff
@@ -0,0 +1,1451 @@
+    Returns:
+        A CommandResult object with a success message, in case of a successful action.
+    """
+    repo = args.get('repo', client.repository)
+    branch_name = args.get('branch_name', '')
+    try:
+        client.branch_delete_request(branch_name, repo)
+        return CommandResults(readable_output=f'The branch {branch_name} was deleted successfully.')
+    except DemistoException as e:
+        m_json = get_exception_message(e)
+        raise Exception(f'{m_json.get("message_arr")} , branch "{m_json.get("error", {}).get("message")}"')
+
+
+def commit_create_command(client: Client, args: Dict) -> CommandResults:
+    """ creates a commit in Bitbucket.
+    Args:
+        client: A Bitbucket client.
+        args: Demisto args.
+    Returns:
+        A CommandResult object with a success message, in case of a successful action.
+    """
+    repo = args.get('repo', client.repository)
+    message = args.get('message', '')
+    branch = args.get('branch', '')
+    file_name = args.get('file_name', None)
+    file_content = args.get('file_content', None)
+    entry_id = args.get('entry_id', None)
+    author_name = args.get('author_name', None)
+    author_email = args.get('author_email', None)
+
+    if not file_name and not entry_id:
+        raise Exception('You must specify either the "file_name" and "file_content" or the "entry_id" of the file.')
+    elif file_name and entry_id:
+        raise Exception('You must specify the "file_name" and "file_content" or the "entry_id" of the file, not both.')
+    elif entry_id:
+        file_path = demisto.getFilePath(entry_id).get('path')
+        file_name = demisto.getFilePath(entry_id).get('name')
+        with open(file_path, 'rb') as f:
+            file_content = f.read()
+    body = {
+        "message": message,
+        "branch": branch,
+        file_name: file_content
+    }
+    if author_name and author_email:
+        body["author"] = f'{author_name} <{author_email}>'
+    elif (author_name and not author_email) or (not author_name and author_email):
+        raise Exception('Both author email and author name should be given')
+    client.commit_create_request(body, repo)
+    return CommandResults(readable_output='The commit was created successfully.')
+
+
+def commit_list_command(client: Client, args: Dict) -> CommandResults:
+    """ Returns a list of all the commits according to the given params.
+    Args:
+        client: A Bitbucket client.
+        args: Demisto args.
+    Returns:
+        A CommandResult object with the list of commits.
+    """
+    repo = args.get('repo', client.repository)
+    file_path = args.get('file_path', None)
+    excluded_branches = args.get('excluded_branches', None)
+    included_branches = args.get('included_branches', None)
+    limit = arg_to_number(args.get('limit')) or 50
+    page = arg_to_number(args.get('page', 1)) or 1
+    page_size = min(limit, 100)
+    excluded_list = argToList(arg=excluded_branches)
```

Comment:
No need for this here - remove those two lines (included and excluded)
Use argToList directly when you get the args (lines 731-732